src/distributions/mod.rs

```diff
@@ -80,6 +82,15 @@ pub trait IndependentSample<Support>: Sample<Support> {
     fn ind_sample<R: Rng>(&self, &mut R) -> Support;
 }
 
+/// DEPRECATED: Use `distributions::uniform` instead.
```

Comment:
Nit (sorry): ``distributions::Uniform`

Reply:
The spelling is as intended, because it addresses the module and not the type.